js/views/conversation_view.js

```diff
@@ -620,7 +627,10 @@
         messageDetail: function(e, data) {
             var view = new Whisper.MessageDetailView({
                 model: data.message,
-                conversation: this.model
+                conversation: this.model,
+                // we pass these in to allow nested panels
+                listenBack: this.listenBack.bind(this),
+                resetPanel: this.resetPanel.bind(this)
```

Comment:
I'm not sure what the best solution is. In the commits where I introduced the detail-expand UI, I included something to the effect of "One less modal", which seemed like a win at the time. Let's not block on it now though since it should be extremely edge casey now with all the profile fetching we do.